Scripts/Runtime/Entities/ProxyDataStream/Data/ProxyInstanceWrapper.cs

```diff
@@ -0,0 +1,75 @@
+using System;
```

Comment:
I don't know why it caught that `ProxyDataID` was renamed to `ProxyInstanceID` but this shows up as a new class. 

Only change is the name